get-release/action.yml

```diff
@@ -1,8 +1,12 @@
-name: 'Download Geode-solutions GitHub Release'
-description: 'Download Geode-solutions Github Release'
+name: 'Download GitHub Release'
+description: 'Download Github Release'
 inputs: 
+  owner:
+    description: 'GitHub owner name (user/organization)'
+    default: 'Geode-solutions'
+    requiered: true
```

Comment:
Maybe not set to true.. So we don't have to change all our previous call to the action